dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/Assertions.java

```diff
@@ -91,16 +91,32 @@ public static void assertHasOnlyErrors( ValidationReport report, ValidationCode.
         assertHasErrors( report, codes.length, codes );
     }
 
+    /**
+     * assertHasWarnings asserts the report contains only given warnings in any
+     * order.
+     *
+     * @param report import report to be asserted on
+     * @param warnings expected warnings
+     */
+    public static void assertHasOnlyWarningsAndNoErrors( ImportReport report, Warning... warnings )
+    {
+        assertAll(
```

Comment:
I think you can just delegate to the `assertHasOnlyWarningsAndNoErrors` taking a `ValidationReport` by calling `report.getValidationReport()`. I used the pattern in the other assertions.